### PR TITLE
Belinda_Next_JS_Sprint_7_386_Fix_Product_Tile_Sizing

### PR DESCRIPTION
**Pull request by MuhammadNSC**
Adjust Tile sizing for product page to all be consistent
Screenshot 2024-05-23 at 1 10 27 PM
Before:

https://github.com/SeattleColleges/belindas-closet-nextjs/assets/117953848/de90edcf-aeb5-4921-af89-91382d567aa9

After:

https://github.com/SeattleColleges/belindas-closet-nextjs/assets/117953848/b39a6e57-5871-4870-960a-02602914ed15

### CODE REVIEW

**Comment by Diego-Cano** [comment]
The screenshot shared clearly shows the discrepancy in the product tiles, particularly with the size and layout of the "Shoes" tiles. The updated code solves the tiles to have uniform sizing, which enhances the overall visual coherence of the product page. This adjustment can be seen in the commit where the tile sizes are modified to align better with the design guidelines.

 I observed that there are conflicts in the package-lock.json and package.json files. These need to be resolved before merging the changes. The consistency in the tile sizes not only enhances the user exp. but also maintains the integrity of the design, which is crucial for user satisfaction. Once the conflicts are resolved and the draft is finalized, this update will significantly improve the UI of the product page.

Also, I found this great link for solving conflicts on github: (https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts).

File: app/category-page/[categoryId]/page.tsx
Comment (gitbiruk2010) [inline app/category-page/[categoryId]/page.tsx:1]: The adjustments to the tile sizes ensure a more consistent and visually appealing layout across different screen sizes. Great work on enhancing the user experience for the product page!

**Comment by tinpham5614** [comment]
The Product Card looks good now.
I'm considering whether we should use columns and rows, so users don't have to scroll through each one.

image

**Comment by brinkbrink** [comment]
Is this still a draft request @MuhammadNSC? Looks like some of this work could be ready to merge after resolving some conflicts in the package-lock.json (should be easy to resolve, just use whatever is from main).

**Review by taylorpapke — APPROVED** [review]
Tile sizing is consistent on all product category pages:
Screenshot 2024-05-28 at 9 10 10 PM

**Comment by MuhammadNSC** [comment]
@brinkbrink I've the issues I was having whit this PR. It can now be merged.
@tinpham5614 having columns will be a good idea especially when there is alot more products on display

**Review by Alzavio — COMMENTED** [review]
Is the height difference across pages a problem? If not, I can approve
image
image

**Comment by MuhammadNSC** [comment]
@Alzavio The height difference is due the the length of the items description

**Review by Alzavio — APPROVED** [review]

**Review by larsswenson — APPROVED** [review]
The tiles' uniformity in size significantly improves improves the UI/UX. I see this was achieved with Grid tags. As more than 2 items are added to each category, will the tiles display in rows/columns, or vertically (as it appears they are now)?

**Review by keiffer213 — CHANGES_REQUESTED** [review]
Let me just put a temporary hold on this before it gets merged

**Review by keiffer213 — APPROVED** [review]

**Pull request merged**
